### ISSUE TITLE
Rule: Methods calling super that should not

### ISSUE THREAD

**Issue by aamctustwo** [realm/SwiftLint#970]
Some of Apple's methods should not call super when overriden, according to their documentation. These methods should be flagged. Example:

```swift
class foo: UIViewController {
  override func loadView() {
    super.loadView() // <- Should be flagged
  }
}
```

**Comment by marcelofabri** [comment]
Do you have other examples? We'd need to keep a list of blacklisted methods, similar to https://github.com/realm/SwiftLint/blob/0.13.2/Source/SwiftLintFramework/Rules/RuleConfigurations/OverridenSuperCallConfiguration.swift#L12

**Comment by aamctustwo** [comment]
@marcelofabri Yep! Readying my PR. :smile:

**Comment by marcelofabri** [comment]
That was fast 🚀

**Comment by aamctustwo** [comment]
@marcelofabri I saw the need for it and saw it was an "easy" PR. So I wrote the PR and went back and raised the issue. PR is #971.
